e2e/playwright/actions/src/tests/create-library.spec.ts

```diff
@@ -23,12 +23,27 @@
  */
 
 import { expect } from '@playwright/test';
-import { Utils, ApiClientFactory, getUserState, test, libraryErrors } from '@alfresco/playwright-shared';
-import { SiteBodyCreate } from '@alfresco/js-api';
+import {
+  Utils,
+  ApiClientFactory,
+  test,
+  libraryErrors,
+  LoginPage,
+  SitesApi,
+  AdfLibraryDialogComponent,
+  DataTableComponent,
+  Breadcrumb,
+  TrashcanApi
+} from '@alfresco/playwright-shared';
+import { logger } from '@alfresco/adf-cli/scripts/logger';
 
-test.use({ storageState: getUserState('hruser') });
 test.describe('Create Libraries ', () => {
   const apiClientFactory = new ApiClientFactory();
+  let sitesApi: SitesApi;
+  let trashcanApi: TrashcanApi;
```

Comment:
the same here - can be moved to beforeEach and mark as const

Reply:
done